web-registry/src/pages/Home/Home.tsx

```diff
@@ -55,6 +57,17 @@ const Home: React.FC = () => {
 
   return (
     <Box sx={{ backgroundColor: 'primary.main' }}>
+      <SEO
+        title={seo?.title || ''}
+        description={seo?.description || ''}
+        imageUrl={seo?.image?.asset?.url || ''} // TODO: specify dimensions here. See: https://www.sanity.io/docs/image-urls
```

Comment:
Just want to make sure you see this - maybe dont’ need to do now though

Reply:
it's not an ideal size, but it does seem to be working [with the browser extension check](https://socialsharepreview.com/browser-extensions) (for ref. for others if they see this). it's actually a bit smaller than the recommended size, so i think it's OK for now (1012x506):

https://cdn.sanity.io/images/jm12rn9t/staging/79a1cedb4c292dbfa50de8a681c67f642806170f-1012x506.png